helmclient.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/spf13/afero"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	schedulingv1alpha1 "k8s.io/api/scheduling/v1alpha1"
```

Comment:
This errored but switching to v1beta1 fixed it. This was with k8s 1.13 and kind.

Reply:
Done